src/main/java/org/terasology/gookeeper/system/BreedingSystem.java

```diff
@@ -77,9 +60,41 @@
     @In
     private NUIManager nuiManager;
 
+    @In
+    private EntityManager entityManager;
+
     private static final Logger logger = LoggerFactory.getLogger(BreedingSystem.class);
     private Random random = new FastRandom();
 
+    /**
+     * This method is called upon when a gooey that is following the player is brought to
```

Comment:
This whole description could be just "This method is called when a gooey steps on a breeding block." Breeding blocks are found only inside love shacks, so you don't need to specify that. Similarly, gooeys won't go into the love shack unless they are brought there, so you don't need to specify that either.